extension.toml

```diff
@@ -2,9 +2,13 @@ id = "kotlin"
 name = "Kotlin"
 version = "0.0.2"
 schema_version = 1
-authors = ["evrsen"]
-description = "Syntax highlighting for Kotlin."
-repository = "https://github.com/evrsen/zed-kotlin"
+authors = ["evrsen", "cholwell"]
```

Comment:
That one I'm not sure of either; I think it makes sense to have just the original creator there, as otherwise we'd have to include each and every person that has ever made a change to this extension.
I think that if anything, CONTRIBUTORS file would be a better fit for the purpose of contribution recognition, if anything.

Reply:
Yeah, I don't have a problem with honoring people's contribution, quite far from that. My only gripe is that maybe CONTRIBUTORS file would be a better fit for that kind of info. 

For what it's worth, `authors` field is optional.

Reply:
We shouldn’t remove authors entirely, though. We want the authors to be visible on the extension page.

Reply:
What action should I take here?

I added my name as it seemed like a significant contribution and the original author isn't interested in making further changes but I'm not sure I care too much

Reply:
Let's leave it as is. Your contribution is indeed significant; I did not mean to undermine that and I'm sorry if it came across as my intent.

Reply:
Not at all 🙂